AuthenticatorBridgeKit/AuthenticatorBridgeItemDataView.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/// A struct for storing **unencrypted** information about items that are shared between the Bitwarden
+/// and Authenticator apps.
+///
+public struct AuthenticatorBridgeItemDataView: Codable, Equatable {
+    // MARK: Properties
+
+    /// Bool indicating if this item is a favorite.
+    public let favorite: Bool
+
+    /// The unique id of the item.
+    public let id: String
+
+    /// The name of the item.
+    public let name: String
+
+    /// The TOTP key used to generate codes.
+    public let totpKey: String?
+
+    /// The username of the Bitwarden account that owns this iteam.
+    public let username: String?
+
+    /// Initialize an `AuthenticatorBridgeItemDataModel` with the values provided.
+    ///
+    /// - Parameters:
+    ///   - favorite: Bool indicating if this item is a favorite.
+    ///   - id: The unique id of the item.
+    ///   - name: The name of the item.
+    ///   - totpKey: The TOTP key used to generate codes.
+    ///   - username: The username of the Bitwarden account that owns this iteam.
+    ///
+    public init(favorite: Bool, id: String, name: String, totpKey: String?, username: String?) {
+        self.favorite = favorite
+        self.id = id
+        self.name = name
+        self.totpKey = totpKey
+        self.username = username
+    }
```

Comment:
⛏️ I think we can remove this as Swift already gives us this initializer for free unless you're planning to do extra logic here in a future PR.
```suggestion
```

Reply:
That one needs to be there so that we can declare in as `public`. I took it off the Model version because we don't want callers (i.e. external to the AuthenticatorBridgeKit library) to init those. But these View ones will be used externally.

Reply:
Ohhh missed the `public` modifier, thanks for clarifying 👍